dev/Microsoft.Process.Environment/EnvironmentManager.h

```diff
@@ -29,6 +29,13 @@ namespace winrt::Microsoft::ProjectReunion::implementation
 
     private:
         Scope m_Scope;
+
+        std::wstring USER_EV_REG_LOCATION = L"Environment";
+        std::wstring MACHINE_EV_REG_LOCATION = L"SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Environment";
+
+        StringMap GetProcessEnvironmentVariables();
```

Comment:
`StringMap Get...() const;` ?

Reply:
const works.  I can add it to my functions.